src/textual/dom.py

```diff
@@ -265,9 +265,12 @@ def get_path(base: Type[DOMNode]) -> str:
             except TypeError:
                 return f"{base.__name__}"
 
-        for tie_breaker, base in enumerate(self._node_bases):
-            css = base.DEFAULT_CSS.strip()
-            if css:
+        default_css = [base.DEFAULT_CSS.strip() for base in self._node_bases]
```

Comment:
I think I see the problem you are trying to solve.

I gather your loop is checking if the `DEFAULT_CSS` is the same as the base class, then skipping it?

Rather than go through the class attribute mechanism, what if we were to look at `__dict__` directly?

```python
base.__dict__get("DEFAULT_CSS")
```

Instead of 

```python
base.DEFAULT_CSS
```

I think the first will return None if the attribute is not explicitly defined on the class, where the second will return the value defined on a base class.

Reply:
That's well thought-of. I tried to come up with a way to check if `DEFAULT_CSS` had been set explicitly in a given class or in its parent classes but I didn't think of `__dict__`. Rewrote the PR so that it uses `__dict__.get(..., "")` to check for the default CSS.